docs/dev-setup/logging-control.md

```diff
@@ -1,78 +1,52 @@
-severity is provided for both users and developers.  There are currently no
-formalized rules for the types of information provided at each severity level,
-however when submitting bug reports the developers may want to see full logs
-down to the DEBUG level.
+All logs are currently directed to `stderr`, and the pretty-printing is currently fixed. However global and module-level control of logging by severity is provided for both users and developers. There are currently no formalized rules for the types of information provided at each severity level, however when submitting bug reports the developers may want to see full logs down to the DEBUG level.
 
-In pretty-printed logs the logging level is indicated both by color and by a
-4-character code, e.g, "ERRO" for ERROR, "DEBU" for DEBUG, etc.  In the
-logging context a _module_ is an arbitrary name (string) given by developers
-to groups of related messages.  In the pretty-printed example below, the
-logging modules "peer", "rest" and "main" are generating logs.
+In pretty-printed logs the logging level is indicated both by color and by a 4-character code, e.g, "ERRO" for ERROR, "DEBU" for DEBUG, etc.  In the logging context a _module_ is an arbitrary name (string) given by developers to groups of related messages. In the pretty-printed example below, the logging modules "peer", "rest" and "main" are generating logs.
 
     16:47:09.634 [peer] GetLocalAddress -> INFO 033 Auto detected peer address: 9.3.158.178:30303
     16:47:09.635 [rest] StartOpenchainRESTServer -> INFO 035 Initializing the REST service...
     16:47:09.635 [main] serve -> INFO 036 Starting peer with id=name:"vp1" , network id=dev, address=9.3.158.178:30303, discovery.rootnode=, validator=true
 
-An arbitrary number of logging modules can be created at runtime, therefore
-there is no "master list" of modules, and logging control constructs can not
-check whether logging modules actually do or will exist.
+An arbitrary number of logging modules can be created at runtime, therefore there is no "master list" of modules, and logging control constructs can not check whether logging modules actually do or will exist.
 
-### PEER
+## peer
 
-The logging level of the `peer` command can be controlled from the command
-line for each invocation using the `--logging-level` flag, for example
+The logging level of the `peer` command can be controlled from the command line for each invocation using the `--logging-level` flag, for example
 
     peer node start --logging-level=debug
 	
-The default logging level for each individual `peer` subcommand can also
-be set in the **openchain.yaml** file. For example the key `logging.peer` sets
-the default level for the `peer` subcommmand.
+The default logging level for each individual `peer` subcommand can also be set in the `core.yaml` file. For example the key `logging.peer` sets the default level for the `peer` subcommmand.
```

Comment:
Some docs such as https://github.com/hyperledger/fabric/blob/master/docs/API/SandboxSetup.md link to files from the doc. I think it's useful and should be the default policy, but open to other opinions.